src/coreclr/inc/jiteeversionguid.h

```diff
@@ -31,11 +31,11 @@
 //
 //////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-constexpr GUID JITEEVersionIdentifier = { /* a7bb194e-4e7c-4850-af12-ea9f30ea5a13 */
-    0xa7bb194e,
-    0x4e7c,
-    0x4850,
-    {0xaf, 0x12, 0xea, 0x9f, 0x30, 0xea, 0x5a, 0x13}
+constexpr GUID JITEEVersionIdentifier = { /* 790de1e5-1426-4ecf-939c-2cd60445c219 */
```

Comment:
@dotnet/jit-contrib what should I do here, should I revert it to the old value before this change?

Reply:
The safest thing to do is to generate a new guid as the comment above describes.

Reply:
Sounds good. I'll do that.